src/Core/Update.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Friendica\Core;
+
+use Friendica\Database\DBStructure;
+
+class Update
+{
+	/**
+	 * Automatic database updates
+	 */
+	public static function run()
+	{
+		$build = Config::get('system', 'build');
+
+		if (empty($build) || ($build > DB_UPDATE_VERSION)) {
```

Comment:
The `DB_UPDATE_VERSION` stays in the `boot.php` because it has to get found easily, hasn't it? That's why I don't add it as a `const` f.e. in `DBStructure` (semantically better understandable).

Reply:
I would move `DB_UPDATE_VERSION` to `index.php`, the goal is to get rid of `boot.php`.

Reply:
But `index.php` isn't called everytime is it?
f.e. I think, the Worker with DB-Upgrade won't execute it, will it? Just to be sure...

Reply:
What about setting it in another `confi/*.ini.php` like `system.ini.php` togetether with the other `define()` in `boot.php` or maybe "just" inside `config.ini.php`.

Reply:
Inside `config.ini.php` is a great idea.